Source/WebKit/UIProcess/ios/WKTouchEventsGestureRecognizer.mm

```diff
@@ -407,7 +448,7 @@ - (BOOL)_hasActiveTouchesForEvent:(UIEvent *)event
     return NO;
 }
 
-- (void)_processTouches:(NSSet *)touches withEvent:(UIEvent *)event type:(WebKit::WKTouchEventType)type
+- (void)_processTouches:(NSSet<UITouch *> *)touches withEvent:(UIEvent *)event type:(WebKit::WKTouchEventType)type
```

Comment:
Do we need this change?

Reply:
technically no, but it is better for type safety